posthog/api/capture.py

```diff
@@ -534,14 +541,25 @@ def get_event(request):
                             sent_at,
                             event_uuid,
                             token,
-                            extra_headers=[("lib_version", lib_version)],
+                            lib_version,
                         )
                     )
 
                 start_time = time.monotonic()
                 for future in futures:
-                    future.get(timeout=settings.KAFKA_PRODUCE_ACK_TIMEOUT_SECONDS - (time.monotonic() - start_time))
-
+                    if future is not None:
+                        future.get(timeout=settings.KAFKA_PRODUCE_ACK_TIMEOUT_SECONDS - (time.monotonic() - start_time))
+    except ValueError as e:
+        with sentry_sdk.push_scope() as scope:
+            scope.set_tag("capture-pathway", "replay")
+            scope.set_tag("token", token)
```

Comment:
what is the token here? is it something we'd know while debugging and searching at sentry?

Reply:
yep, or rather we'd know it debugging in posthog
it lets us see what team was affected

Reply:
will update tag name for the future traveller